lstchain/image/muon/muon_analysis.py

```diff
@@ -125,13 +127,29 @@ def fit_muon(x, y, image, geom, tailcuts):
     image_clean: `np.ndarray`
         Image after cleaning
     """
+    
+    if tailcuts == None:
+        #we want to quantize the noise of the image. To do so, we will use the negative Q cumulative distribution.
+        negative_Q=np.sort(image[image <= 0])
+
+        hist, bins = np.histogram(negative_Q,range=(-15,0),bins=30)
+        bins=bins[:-1]
```

Comment:
This modification of "bins" seems unnecessary

Reply:
transferred to #1112